app/views/avo/actions/show.html.erb

```diff
@@ -51,7 +51,8 @@
             size: :sm,
             data: {
               target: :submit_action,
-            } do %>
+              action_target: "submit"
```

Comment:
Do we really need this target?

```suggestion
```

Reply:
I think this came from the car when I resolved the conflict